debian/control

```diff
@@ -24,6 +24,7 @@ Build-Depends:
  libglib2.0-dev (>= 2.54.0),
  libgsystem-dev,
  libjson-glib-dev (>= 1.2.6-1endless2),
+ libmogwai-schedule-client-0-dev,
```

Comment:
```
/usr/bin/ld: cannot find -lmogwai-schedule-client-0
collect2: error: ld returned 1 exit status
Makefile:753: recipe for target 'eos-updater' failed
```

Reviewing the [build log](https://ci.endlessm-sf.com/job/obs-eos-updater-pr/79/consoleFull) it looks like the -dev package may not depend on the shared library package? I think it is conventional to have this dependency.

Reply:
That seems reasonable. https://github.com/endlessm/mogwai/pull/22 opened.

Reply:
I suspect I was expecting the `${shlibs:Depends}` dependency for `eos-updater` itself to pull `libmogwai-schedule-client-0` in; but it will actually only do that when you install the `eos-updater` package — not at build time, right?

Reply:
Yes, I think that's right. I was surprised to rediscover this recently.